src/main/java/fact/utils/ElementwiseMultiplyDoubleArray.java

```diff
@@ -0,0 +1,46 @@
+package fact.utils;
+
+import stream.Data;
+import stream.Processor;
+import stream.annotations.Parameter;
+import fact.features.singlePulse.timeSeriesExtraction.ElementWise;
```

Comment:
I think it's clear now that this class belongs in a more general package than `singlePulse`